inference-engine/tests/functional/plugin/shared/include/subgraph_tests/two_fake_quantize_to_fullyconnected.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <tuple>
+#include <vector>
+#include <string>
+#include <memory>
+
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+
+typedef std::tuple<
+        std::vector<size_t>, // levels
+        std::vector<size_t>, // const inputs shape
+        std::vector<float>,  // fake quantize inputLow, inputHigh, outputLow, outputHigh or empty for random
+        std::vector<float>   // input generator data: low, high, resolution
+> fqSpecificParams;
+typedef std::tuple<
+        fqSpecificParams,
+        InferenceEngine::Precision,        // Net precision
+        InferenceEngine::Precision,        // Input precision
+        InferenceEngine::Precision,        // Output precision
+        InferenceEngine::Layout,           // Input layout
+        InferenceEngine::Layout,           // Output layout
+        InferenceEngine::SizeVector,       // Input shapes
+        LayerTestsUtils::TargetDevice,     // Device name
+        std::pair<std::string, std::map<std::string, std::string>>, // Additional backend configuration and alis name to it
+        bool
+> fqSubgraphTestParamsSet;
+namespace LayerTestsDefinitions {
+
+
+class FakeQuantizeSubgraphTest : public testing::WithParamInterface<fqSubgraphTestParamsSet>,
+                              virtual public LayerTestsUtils::LayerTestsCommon {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<fqSubgraphTestParamsSet> obj);
+    InferenceEngine::Blob::Ptr GenerateInput(const InferenceEngine::InputInfo &info) const override;
+protected:
+    void SetUp() override;
+    void UpdateSeed();
+
+protected:
+    float inputDataMin        = 0.0;
+    float inputDataMax        = 10.0;
```

Comment:
It would be good to have tests checking cases with max input much smaller and much bigger than 10.

Reply:
These parameters can be set from the test arguments. That is, you can add any limits to check the test performance.